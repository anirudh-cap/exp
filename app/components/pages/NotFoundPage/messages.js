/*
 * NotFoundPage Messages
 *
 * This contains all the text for the NotFoundPage container.
 */
import { defineMessages } from 'react-intl';

export const scope = 'loyaltyPlus.components.containers.NotFoundPage';

export default defineMessages({
  header: {
    id: `${scope}.header`,
    defaultMessage: 'Looks like you are lost!',
  },
});
