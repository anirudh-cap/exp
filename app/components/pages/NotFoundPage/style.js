import { css } from 'styled-components';

export default css`
  .big-owl {
    background-color: red;
    height: 600px;
    width: 600px;
  }

  .red-bg {
    background-color: red;
  }
`;
