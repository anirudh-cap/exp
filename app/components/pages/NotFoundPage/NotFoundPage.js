import React from 'react';
import { FormattedMessage } from 'react-intl';
import messages from './messages';
import PageTemplate from '../../templates/PageTemplate';
import Heading from '../../atoms/Heading';

export const NotFoundPage = () => (
  <PageTemplate>
    <Heading>
      <FormattedMessage {...messages.header} />
    </Heading>
  </PageTemplate>
);

export default NotFoundPage;
