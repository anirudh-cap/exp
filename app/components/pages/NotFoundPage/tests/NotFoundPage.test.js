import React from 'react'; // eslint-disable-line no-unused-vars
import { shallow } from 'enzyme';

import NotFoundPage from '../NotFoundPage';

describe('<NotFoundPage />', () => {
  it('should render the page message', () => {
    const renderedComponent = shallow(<NotFoundPage />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
