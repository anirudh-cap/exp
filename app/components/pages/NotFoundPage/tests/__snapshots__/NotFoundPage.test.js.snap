// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should render the page message 1`] = `
<PageTemplate>
  <withStyles__StyledComponent
    className=""
  >
    <FormattedMessage
      defaultMessage="Looks like you are lost!"
      id="loyaltyPlus.components.containers.NotFoundPage.header"
      values={Object {}}
    />
  </withStyles__StyledComponent>
</PageTemplate>
`;
